src/App.jsx

```diff
@@ -7,14 +7,28 @@ import TodoList from './TodoList'
 import './AddTodoForm'
 import AddTodoForm from './AddTodoForm'
 
+const semiPersistentState = () => {
```

Comment:
according to the react documentation, a custom hook should start with the work `use` 
see here https://react.dev/learn/reusing-logic-with-custom-hooks#hook-names-always-start-with-use

```suggestion
const useSemiPersistentState = () => {
```